apiserver/facades/client/bundle/bundle.go

```diff
@@ -8,47 +8,52 @@ import (
 	"strings"
 
 	"github.com/juju/bundlechanges"
+	"github.com/juju/description"
 	"github.com/juju/errors"
 	"github.com/juju/loggo"
 	"gopkg.in/juju/charm.v6"
+	names "gopkg.in/juju/names.v2"
 
 	"github.com/juju/juju/apiserver/common"
 	"github.com/juju/juju/apiserver/facade"
 	"github.com/juju/juju/apiserver/params"
 	"github.com/juju/juju/constraints"
 	"github.com/juju/juju/core/devices"
-	"github.com/juju/juju/state"
 	"github.com/juju/juju/storage"
 )
 
-// NewFacade provides the required signature for facade registration.
-func NewFacade(_ *state.State, _ facade.Resources, auth facade.Authorizer) (Bundle, error) {
-	return NewBundle(auth)
+type Facade struct {
```

Comment:
No need for a new struct, just use the existing bundleAPI struct

Reply:
done